graphql/src/main/java/io/jmix/graphql/schema/Types.java

```diff
@@ -24,7 +24,11 @@
         GT(PropertyCondition.Operation.GREATER),
         GTE(PropertyCondition.Operation.GREATER_OR_EQUAL),
         LT(PropertyCondition.Operation.LESS),
-        LTE(PropertyCondition.Operation.LESS_OR_EQUAL);
+        LTE(PropertyCondition.Operation.LESS_OR_EQUAL),
+        _cont(PropertyCondition.Operation.CONTAINS),
+        _ncont(PropertyCondition.Operation.NOT_CONTAINS),
+        _stw(PropertyCondition.Operation.STARTS_WITH),
+        _enw(PropertyCondition.Operation.ENDS_WITH);
```

Comment:
we decide use full names of operators
_contains, _notContains, _startsWith, _endsWith